packages/sync/src/modules/Comments.php

```diff
@@ -177,6 +178,16 @@ public function get_whitelisted_comment_types() {
 		);
 	}
 
+	public function only_allow_white_listed_comment_types( $args ) {
+			$comment = $args[1];
```

Comment:
Excessive indentation here:

```suggestion
		$comment = $args[1];
```